godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/Godot.kt

```diff
@@ -95,7 +95,6 @@ object Godot {
 
     internal fun printError(description: String, function: String, file: String, line: Int, exception: Exception) {
```

Comment:
now the exception is unused. I would rename the method or at least print the stacktrace (maybe append it to the description?)